pkg/basicstation/cups/messages.go

```diff
@@ -84,6 +90,17 @@ func TokenCredentials(trust *x509.Certificate, authorization string) ([]byte, er
 	out = append(out, trust.Raw...)
 	// TODO: Refactor when client side TLS is supported https://github.com/TheThingsNetwork/lorawan-stack/issues/137
 	out = append(out, emptyClientCert...)
+	token := authorization
+	if strings.HasPrefix(authorization, "Bearer") {
+		t := strings.Split(authorization, " ")
+		if len(t) != 2 {
+			return nil, errInvalidAuthToken.WithAttributes("token", authorization)
+		}
+		token = t[1]
+	}
```

Comment:
Why do we need to strip the `Bearer` part? What if someone wants to connect their gateway to a different network that does require `Bearer`?

Reply:
We don't. We just strip it for validating the token. `authorization` is the original variable that's used unchanged if the validation is successful.